### DISCUSSION TITLE
When Xamarin.Forms will support build for Linux ?

### DISCUSSION THREAD

**Discussion by redradist** [xamarin/Xamarin.Forms#11888] [General]
I am wondering when Xamarin.Forms will support Linux build ?
Will it be available in .NET 5 ?

**Comment by EmilAlipiev** [comment]
there is a plan for Mau project as you may have heard it. It will run in Visual Studio code so you can use with Linux but not in xamarin.forms

**Reply by redradist** [reply]
Okay, it is cool for MAUI, but will be build for Linux available for .NET 5 ?

**Reply by EmilAlipiev** [reply]
yea i guess it will be with .net core 5. you can follow the progress and news on Maui github repo
